models/userStatus.js

```diff
@@ -330,98 +330,108 @@ const updateStatusOnTaskCompletion = async (userId) => {
 const massUpdateIdleUsers = async (users) => {
```

Comment:
why massUpdateIdleUsers can we do only updateIdleUsers

Reply:
Since now we are updating both idle and active users. I updated the name to batchUpdateUsersStatus as updateUserStatus already existed.